islandora_pathauto.module

```diff
@@ -187,6 +190,13 @@ function islandora_pathauto_pathauto_bulkupdate(&$context) {
       $filter = "sameTerm(?model, <info:fedora/$filter>)";
     }
     $sandbox['filters']['model'] = format_string('FILTER(!models)', array('!models' => implode(' || ', $model_filters)));
+    if (variable_get('islandora_namespace_restriction_enforced', FALSE)) {
```

Comment:
The logic seems a lot more complex than the one already tested and proved working at `function islandora_islandora_basic_collection_get_query_filters()` which  goes like this 
```PHP 
$enforced = variable_get('islandora_namespace_restriction_enforced', FALSE);
  if ($enforced) {
    $namespace_array = islandora_get_allowed_namespaces();
    $namespace_sparql = implode('|', $namespace_array);
    return array('islandora_namespace_restrictions' => format_string('regex(str(?object), "info:fedora/(!namespaces):")', array('!namespaces' => $namespace_sparql)));
  }
````
Any reason for go for this way instead?